docs/02-app/02-api-reference/05-next-config-js/output.mdx

```diff
@@ -41,6 +41,7 @@ Additionally, a minimal `server.js` file is also output which can be used instea
 >
 > - `next.config.js` is read during `next build` and serialized into the `server.js` output file. If the legacy [`serverRuntimeConfig` or `publicRuntimeConfig` options](/docs/app/api-reference/next-config-js/runtime-configuration) are being used, the values will be specific to values at build time.
 > - If your project uses [Image Optimization](/docs/app/building-your-application/optimizing/images) with the default `loader`, you must install `sharp` as a dependency:
+> - If your project needs alternative port or hostname for listening, you can define `PORT` and `HOSTNAME` environment variables, before running `server.js`. For example, `PORT=3000 HOSTNAME=0.0.0.0 node server.js`.
```

Comment:
```suggestion
> - If your project needs alternative port or hostname for listening, you can define `PORT` and `HOSTNAME` environment variables, before running `server.js`. For example, `PORT=3000 HOSTNAME=localhost node server.js`.
```

Lets use the defaults here and the reader can decide to change if they want

Reply:
Ok 😢